roles/wazuh/ansible-wazuh-manager/tasks/main.yml

```diff
@@ -1,17 +1,38 @@
 ---
 
 - name: "Install dependencies"
-  package:
-    name:
-      - unzip
-      - openssl
-      - tar
-      - curl
-    state: present
-  register: package_status
-  until: "package_status is not failed"  
-  retries: 10
-  delay: 10
+  block:
+    - name: "Install common dependencies"
+      package:
+        name:
+          - unzip
+          - openssl
+          - tar
+        state: present
+      register: package_status
+      until: "package_status is not failed"  
+      retries: 10
+      delay: 10
+
+    - name: "Install curl"
+      package:
+        name: curl
+        state: present
+      when: ansible_distribution != "Amazon" or (ansible_distribution == "Amazon" and ansible_distribution_version != "2023")
```

Comment:
Can this condition be changed from:
`ansible_distribution != "Amazon" or (ansible_distribution == "Amazon" and ansible_distribution_version != "2023")`
To:
`ansible_distribution != "Amazon" and ansible_distribution_version != "2023"`

Reply:
Fixed in [d2b5b62](https://github.com/wazuh/wazuh-ansible/pull/1338/commits/d2b5b62bcefe859a497c8b65357b1e039297448f)